paddle/fluid/framework/operator.cc

```diff
@@ -457,6 +460,8 @@ const Tensor* ExecutionContext::Input<Tensor>(const std::string& name) const {
 template <>
 const std::vector<const Tensor*> ExecutionContext::MultiInput<Tensor>(
     const std::string& name) const {
+  LogVarUsageIfUnusedVarCheckEnabled(name);
```

Comment:
Call this function with `if (FLAGS_enable_unused_var_check)`?

Reply:
To reduce code duplication,  the `if`  statement is inside `LogVarUsageIfUnusedVarCheckEnabled`. Is it good to move it out?